src/actions/operator.ts

```diff
@@ -280,6 +280,12 @@ export class YankOperator extends BaseOperator {
 
     Register.put(text, vimState, this.multicursorIndex);
 
+    if (vimState.currentMode === ModeName.Visual || vimState.currentMode === ModeName.VisualLine) {
+      end = new Position(end.line, end.character - 1);
```

Comment:
What is `end.character` = 0?

Reply:
Higher up in the same method it's `end = new Position(end.line, end.character + 1);`. I just reset it.

Reply:
As this is specific to the `if` block, maybe create a temp variable? Nobody is using `end` after this, but this would be prone to bugs if we modified it's value and left it like this.

Reply:
Yep, good idea.

Reply:
b854385 fixes this.